app/src/androidTest/java/uk/gov/onelogin/signOut/ui/SignOutScreenKtTest.kt

```diff
@@ -7,26 +7,39 @@ import androidx.compose.ui.test.performClick
 import dagger.hilt.android.testing.BindValue
 import dagger.hilt.android.testing.HiltAndroidTest
 import dagger.hilt.android.testing.UninstallModules
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.test.runTest
 import org.junit.Before
 import org.junit.Test
 import org.mockito.Mockito.mock
+import org.mockito.kotlin.any
 import org.mockito.kotlin.never
 import org.mockito.kotlin.verify
 import org.mockito.kotlin.whenever
 import uk.gov.android.onelogin.R
+import uk.gov.android.wallet.sdk.WalletSdk
 import uk.gov.onelogin.TestCase
 import uk.gov.onelogin.signOut.SignOutModule
 import uk.gov.onelogin.signOut.domain.SignOutError
 import uk.gov.onelogin.signOut.domain.SignOutUseCase
+import uk.gov.onelogin.wallet.DeleteWalletDataUseCase
+import uk.gov.onelogin.wallet.WalletModule
 
 @HiltAndroidTest
 @UninstallModules(
-    SignOutModule::class
+    SignOutModule::class,
+    WalletModule::class
 )
 class SignOutScreenKtTest : TestCase() {
```

Comment:
_I know this isn't something you have done_
Test classes should be named <ClassName>Test, the Kt bit gets added by mistake via Android Studio. Do watch out for this as you make tests, and it is appreciated, but not essential, if you could rename them when you come across them while making changes (should be in a separate commit just for the renaming if you do please). 
Though leave it alone if not making any changes to that file